self-tests.yml

```diff
@@ -867,6 +881,7 @@ buildvariants:
       GOROOT: /opt/golang/go1.24
     tasks:
       - name: generate-lint
+      - name: lint-markdown
```

Comment:
We pick up all ".linter" in another bv, I don't think we need it in this one too (and it might be confusing cause the bv is currently only the generator and generated lint tasks)